newsletter-2021-02.html

```diff
@@ -0,0 +1,92 @@
+<h1>tedana February 2021 Update</h1>
+Since our late December newsletter, we have created a new stable release available on PyPI!
+You can upgrade your local installation to the latest 0.0.9 version by running <xmp>pip install -U tedana</xmp>.
```

Comment:
At least on Chrome, after downloading the raw HTML file, `<xmp>pip install -U tedana</xmp>` ends up on its own line, and the following period shows up before "We encourage" on its own line.

I don't do much with HTML, so maybe this isn't an issue?

Reply:
I could just remove the period. I missed that it's on its own line. Thoughts?

I don't do much HTML either, it's just what tinyletter uses for formatting.